maniloo/spark_test_task
# Import products from csv

#### Why you made this change:
I did it to add a possibility to imports products from CSV files

#### How the change addresses the need:
New controller, service, and the job is added make import possible

#### Testing procedure:
1. Log in to admin panel
2. Visit `admin/products_import/new` path to create new import
3. Choose CSV file
4. Submit form
5. You should see a green message that imports start
6. Visit `admin/products` path to see imported products
